common/src/main/java/com/wynntils/models/character/type/SetInfo.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.character.type;
+
+import net.minecraft.world.item.ItemStack;
+
+public class SetInfo {
+    private final int wynncraftCount;
+    private int trueCount = 0;
+    private final ItemStack relevantItem;
+
+    public SetInfo(int wynncraftCount, ItemStack relevantItem) {
+        this.wynncraftCount = wynncraftCount;
+        this.relevantItem = relevantItem;
+    }
+
+    public int getWynncraftCount() {
```

Comment:
Is Wynncraft count counting rings twice, or the other way around? Or do I misunderstand this? A comment describing the differences between the "count" functions would be good.

Reply:
If you have the same ring on both slots it is only counted once